Message-Id: <af426bb880140c2941366a3a84fd11a714b9c778.1336068799.git.agx@sigxcpu.org>
In-Reply-To: <727a1c60fdc68e24b2f457f5b7e3611b2aae7a7b.1336068799.git.agx@sigxcpu.org>
References: <5909887366833c92178e3ed650b87d666f9f50c1.1336068799.git.agx@sigxcpu.org>
	<56db9d711b7615ece3848c57f71f362f0e5f4fea.1336068799.git.agx@sigxcpu.org>
	<727a1c60fdc68e24b2f457f5b7e3611b2aae7a7b.1336068799.git.agx@sigxcpu.org>
From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 2 Apr 2011 10:37:15 +0200
Subject: Brand de

---
 de/locale/de/calendar/calendar.properties   |    4 ++--
 de/locale/de/calendar/migration.properties  |    2 +-
 de/locale/de/calendar/timezones.properties  |    2 +-
 de/locale/de/lightning/lightning.dtd        |    2 +-
 de/locale/de/lightning/lightning.properties |    6 +++---
 5 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/de/locale/de/calendar/calendar.properties b/de/locale/de/calendar/calendar.properties
index 542fe8b..1a27268 100644
--- a/de/locale/de/calendar/calendar.properties
+++ b/de/locale/de/calendar/calendar.properties
@@ -381,8 +381,8 @@ pw_change2empty_in_fips_mode=Sie sind derzeit im FIPS-Modus. FIPS benötigt ein
 pw_change_failed_title=Ändern des Passworts fehlgeschlagen
 
 # Check for Updates     (SUNBIRD_ONLY)
-updatesItem_default=Sunbird aktualisieren…
-updatesItem_defaultFallback=Sunbird aktualisieren…
+updatesItem_default=Iceowl aktualisieren…
+updatesItem_defaultFallback=Iceowl aktualisieren…
 updatesItem_downloading=%S wird heruntergeladen…
 updatesItem_downloadingFallback=Update wird heruntergeladen…
 updatesItem_resume=Herunterladen von %S fortsetzen…
diff --git a/de/locale/de/calendar/migration.properties b/de/locale/de/calendar/migration.properties
index ba97110..bd7e3c2 100644
--- a/de/locale/de/calendar/migration.properties
+++ b/de/locale/de/calendar/migration.properties
@@ -43,4 +43,4 @@ migrationTitle = %1$S: Daten-Import
 migrationDescription=%1$S kann Kalenderdaten von vielen bekannten Programmen importieren. Es wurden Daten von folgenden Programmen auf Ihrem Computer gefunden. Bitte wählen Sie aus, von welchen Programmen Sie Daten importieren möchten.
 finished = Komplett
 disableExtTitle = Inkompatible Erweiterung gefunden
-disableExtText = Sie haben die alte Mozilla Kalendererweiterung installiert, welche mit Lightning nicht kompatibel ist. Sie wird deaktiviert und %1$S wird neu gestartet.
+disableExtText = Sie haben die alte Mozilla Kalendererweiterung installiert, welche mit Iceowl nicht kompatibel ist. Sie wird deaktiviert und %1$S wird neu gestartet.
diff --git a/de/locale/de/calendar/timezones.properties b/de/locale/de/calendar/timezones.properties
index fb05138..549f6ac 100644
--- a/de/locale/de/calendar/timezones.properties
+++ b/de/locale/de/calendar/timezones.properties
@@ -41,7 +41,7 @@
 
 # extension:
 extensions.calendar-timezones@mozilla.org.name=Zeitzonendefinitionen für Mozilla Calendar
-extensions.calendar-timezones@mozilla.org.description=Von Sunbird und Lightning benötigte Zeitzonendefinitionen
+extensions.calendar-timezones@mozilla.org.description=Von Sunbird und Iceowl benötigte Zeitzonendefinitionen
 extensions.calendar-timezones@mozilla.org.creator=Mozilla Calendar Project
 
 pref.timezone.floating=Lokale Zeit
diff --git a/de/locale/de/lightning/lightning.dtd b/de/locale/de/lightning/lightning.dtd
index 26ee6a6..425abb3 100644
--- a/de/locale/de/lightning/lightning.dtd
+++ b/de/locale/de/lightning/lightning.dtd
@@ -46,7 +46,7 @@
   -->
    
 <!-- Tools menu -->
-<!ENTITY lightning.taskLabel "Lightning">
+<!ENTITY lightning.taskLabel "Iceowl">
 
 <!-- New menu popup in File menu -->
 <!ENTITY lightning.menupopup.new.event.label        "Termin…">
diff --git a/de/locale/de/lightning/lightning.properties b/de/locale/de/lightning/lightning.properties
index d870348..ec763d8 100644
--- a/de/locale/de/lightning/lightning.properties
+++ b/de/locale/de/lightning/lightning.properties
@@ -37,12 +37,12 @@
 # ***** END LICENSE BLOCK *****
 
 # addon description:
-extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.name=Lightning
+extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.name=Iceowl
 extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.description=Ein integrierter Kalender für Thunderbird
 extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.creator=Mozilla Kalender-Projekt
 
 # Lightning branding
-brandShortName=Lightning
+brandShortName=Iceowl
 
 # Task mode title
 taskModeApplicationTitle=Aufgaben
@@ -71,7 +71,7 @@ imipBarRequestText=Diese Nachricht enthält eine Termineinladung.
 imipBarUpdateText=Diese Nachricht enthält eine Aktualisierung für einen vorhandenen Termin.
 imipBarAlreadyProcessedText=Diese Nachricht enthält einen Termin, der bereits verarbeitet wurde.
 imipBarReplyText=Diese Nachricht enthält eine Antwort auf eine Einladung.
-imipBarUnsupportedText=Diese Nachricht enthält einen Termin, der mit dieser Version von Lightning nicht verarbeitet werden kann.
+imipBarUnsupportedText=Diese Nachricht enthält einen Termin, der mit dieser Version von Iceowl nicht verarbeitet werden kann.
 imipBarProcessingFailed=Bearbeiten der Nachricht fehlgeschlagen. Status: %1$S.
 imipAcceptInvitation.label=Akzeptieren
 imipCancelInvitation.label=Löschen
